test/ValidityTest.hs

```diff
@@ -111,31 +110,31 @@ operationStripDir = do
                 stripDir parent (parent </> child) == Just child
 
      it "produces a valid path on when passed a valid absolute file paths" $ do
-        producesValidsOnValids2 (stripDir @Maybe @Abs @File)
+             producesValidsOnValids2 (stripDir :: Path Abs Dir -> Path Abs File -> Maybe (Path Rel File))
```

Comment:
uhg, it's just because I have to use another editor to do these edits because I normally use hindent.
I will try to fix these, but at some point we should start using a zift script, or something.